examples/sinatra_example/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.7'
+
+services:
+  postgres:
+    image: cimg/postgres:14.2
```

Comment:
suggestion: postgres:14.6-alpine, because cimg/postgres weighs almost 1GB, whereas alpine is around 100mb

Reply:
wowzers, yea, I'll change that.